src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -433,7 +448,8 @@ public void sleepSeconds (int seconds) {
                 TravelTimeSurfaceTask timeSurfaceTask = (TravelTimeSurfaceTask) request;
                 if (timeSurfaceTask.getFormat() == TravelTimeSurfaceTask.Format.GRID) {
                     // Return raw byte array representing grid to caller, for return to client over HTTP.
-                    byteArrayOutputStream.write(oneOriginResult.timeGrid.writeGrid());
+                    oneOriginResult.timeGrid.writeGridToStream(new LittleEndianDataOutputStream(byteArrayOutputStream));
```

Comment:
The grid format itself is an array of little-endian ints, so shouldn't the LittleEndianDataOutputStream usage be encapsulated entirely within this method that writes the grid format?